modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -122,6 +118,10 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
     const newWrappers: Element[] = [];
     let contentEditable = true;
 
+    const isWrappingBlockFormat = FormatUtils.isBlockFormat(format) && format.wrapper;
+    const isNonWrappingBlockFormat = FormatUtils.isBlockFormat(format) && !format.wrapper;
```

Comment:
Should these be in `FormatUtils` instead? They definitely seem out of place here is all.

Reply:
Good idea. Will do